README.md

```diff
@@ -175,14 +175,18 @@ of multiple BLAS operations.
 
 ## API description
 
-This section references all the supported operations and their interface.
+This section references all the supported operations and their interface. The library follows the [oneAPI MKL BLAS specification](https://spec.oneapi.io/versions/latest/elements/oneMKL/source/domains/blas/blas.html) as reference for the api. We have support for both USM and Buffer api, however the group apis for USM are not supported. We don't support mixing USM and Buffer arguments together to compile the library, and instead stick to the aformentioned reference specification.
```

Comment:
It would be better to add a couple of newlines for padding it with the rest of the text.

Reply:
Addressed this in the latest commit.